app/helpers/logging_helper.rb

```diff
@@ -0,0 +1,34 @@
+module LoggingHelper
```

Comment:
To be honest, I'm not even certain that this thread local approach is safe. Puma uses event machine (I think) and I'm not 100% certain that an HTTP request is always processed inside one single thread.